istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -211,5 +211,13 @@ data:
       envoyAccessLogService:
         address: {{ .Values.global.proxy.envoyAccessLogService.host }}:{{ .Values.global.proxy.envoyAccessLogService.port }}
     {{- end}}
+
+    {{- if .Values.pilot.enableCertificateController }}
```

Comment:
Same as above: a 'certificates' section under values.global, no need for an 'enable' API.

Reply:
Done: addressed in https://github.com/istio/installer/pull/424; please review it.